package.json

```diff
@@ -68,17 +68,17 @@
     "lodash": "^4.17.19",
     "mapeo-offline-map": "^2.0.0",
     "mapeo-schema": "^2.0.2",
-    "nodejs-mobile-react-native": "^0.6.1",
+    "nodejs-mobile-react-native": "^0.6.3",
```

Comment:
ah yeah i forgot about that 🤦‍♂️ but yes i can look into it.

do you know which commit in https://github.com/nodejs-mobile/nodejs-mobile-react-native introduces the new node version being used? wondering if downgrading to `0.6.2` and adding the appropriate patches would suffice  (this assumes that 0.6.3 is the problematic release, moot otherwise)

Reply:
ah i see that 0.6.2 introduces the new node version. disregard previous comment then